test/adapters/net_http_persistent_test.rb

```diff
@@ -10,8 +10,10 @@ def setup
         if defined?(Net::HTTP::Persistent)
           # work around problems with mixed SSL certificates
           # https://github.com/drbrain/net-http-persistent/issues/45
-          http = Net::HTTP::Persistent.new('Faraday')
-          http.ssl_cleanup(4)
+          if Net::HTTP::Persistent::instance_methods.include?(:ssl_cleanup)
+            http = Net::HTTP::Persistent.new('Faraday')
+            http.ssl_cleanup(4)
+          end
```

Comment:
right, looks like you have to use this `reuse_ssl_sessions`. Did you spot also what the default value is?